### PR TITLE
Start documenting prioritary tasks.

### PR DESCRIPTION
**Pull request by AlbertoGP**

### CODE REVIEW

**Review by MBeijer — APPROVED** [review]

**Comment by AlbertoGP** [comment]
The content is based on the discussion today with Gunnar and others in the Discord channel: https://discord.com/channels/726870908568076380/730698753513750539/808265391184412692

File: docs/tasks/GFX-subsystem.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+4. Open a few folder windows and move them around a little, see how fast it is.
+
+High priority bugs for the GFX subsystem are listed under:
+https://github.com/ApolloTeam-dev/AROS/issues/8
+
+## Compiling and testing
+Workbook / `workbench.library` is in ROM.
+
+- With the [`master-new`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
+- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/v4-alynna/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it. There are some improvements pending merge at: [Some mkapollo.sh enhancements #27](https://github.com/ApolloTeam-dev/AROS/pull/27)
```
Comment (MBeijer) [inline docs/tasks/GFX-subsystem.md]: mkapollo is merged to master now
```suggestion
- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/master/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it. 
```

File: docs/tasks/IDE.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Documentation from the [Amiga Developer CD v2.1 (OS 3.5)](http://amigadev.elowar.com/):
+
+- “[Amiga® RKM Libraries: Exec Device I/O](http://amigadev.elowar.com/read/ADCD_2.1/Libraries_Manual_guide/node0299.html)”
+- “[Amiga® RKM Devices: Contents](http://amigadev.elowar.com/read/ADCD_2.1/Devices_Manual_guide/node0000.html)”
+
+## Compiling and testing
+IDE is in ROM.
+
+- With the [`master-new`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
+- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/v4-alynna/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it. There are some improvements pending merge at: [Some mkapollo.sh enhancements #27](https://github.com/ApolloTeam-dev/AROS/pull/27)
```
Comment (MBeijer) [inline docs/tasks/IDE.md]:
```suggestion
- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/v4-alynna/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it.
```

File: docs/tasks/IDE.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+Documentation from the [Amiga Developer CD v2.1 (OS 3.5)](http://amigadev.elowar.com/):
+
+- “[Amiga® RKM Libraries: Exec Device I/O](http://amigadev.elowar.com/read/ADCD_2.1/Libraries_Manual_guide/node0299.html)”
+- “[Amiga® RKM Devices: Contents](http://amigadev.elowar.com/read/ADCD_2.1/Devices_Manual_guide/node0000.html)”
+
+## Compiling and testing
+IDE is in ROM.
+
+- With the [`master-new`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```
Comment (MBeijer) [inline docs/tasks/IDE.md]:
```suggestion
- With the [`master`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```
master-new is now renamed master

File: docs/tasks/GFX-subsystem.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+3. Click on some HD icon so that the window appears.
+4. Open a few folder windows and move them around a little, see how fast it is.
+
+High priority bugs for the GFX subsystem are listed under:
+https://github.com/ApolloTeam-dev/AROS/issues/8
+
+## Compiling and testing
+Workbook / `workbench.library` is in ROM.
+
+- With the [`master-new`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```
Comment (MBeijer) [inline docs/tasks/GFX-subsystem.md]:
```suggestion
- With the [`master`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```

File: docs/tasks.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- [GFX subsystem](tasks/GFX-subsystem.md)
+
+- [SD Card R/W](tasks/SD-card.md)
+
+You can also take a look at the “Open Bug List“:
+https://github.com/ApolloTeam-dev/AROS/issues/8
+
+The general build instructions are at:
+https://github.com/ApolloTeam-dev/AROS/wiki/Building
+Do not forget the `--recursive` option with `git clone`!
```
Comment (MBeijer) [inline docs/tasks.md]:
```suggestion
**Do not forget the `--recursive` option with `git clone`!**
```

File: docs/tasks/GFX-subsystem.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
+# GFX subsystem
+
+Source code:
```
Comment (MBeijer) [inline docs/tasks/GFX-subsystem.md]:
```suggestion
## Source code
```

File: docs/tasks/GFX-subsystem.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+- [`rom/graphics/`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/rom/graphics/)
+
+Documentation from AROS:
```
Comment (MBeijer) [inline docs/tasks/GFX-subsystem.md]:
```suggestion
## Documentation from AROS
```

File: docs/tasks/GFX-subsystem.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
+# GFX subsystem
+
+Source code:
+
+- [`rom/graphics/`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/rom/graphics/)
```
Comment (MBeijer) [inline docs/tasks/GFX-subsystem.md]:
```suggestion
- [`rom/graphics/`](https://github.com/ApolloTeam-dev/AROS/tree/master/rom/graphics/)
```

File: docs/tasks/IDE.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+# IDE controller
+
+Source code:
+
+- [`rom/devs/ata/`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/rom/devs/ata)
```
Comment (MBeijer) [inline docs/tasks/IDE.md]:
```suggestion
- [`rom/devs/ata/`](https://github.com/ApolloTeam-dev/AROS/tree/master/rom/devs/ata)
```

File: docs/tasks/IDE.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+- [`arch/m68k-all/`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/arch/m68k-all)
```
Comment (MBeijer) [inline docs/tasks/IDE.md]:
```suggestion
- [`arch/m68k-all/`](https://github.com/ApolloTeam-dev/AROS/tree/master/arch/m68k-all)
```

File: docs/tasks/IDE.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+- [`arch/m68k-amiga/`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/arch/m68k-amiga)
```
Comment (MBeijer) [inline docs/tasks/IDE.md]:
```suggestion
- [`arch/m68k-amiga/`](https://github.com/ApolloTeam-dev/AROS/tree/master/arch/m68k-amiga)
```

File: docs/tasks/SD-card.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+Documentation from the [Amiga Developer CD v2.1 (OS 3.5)](http://amigadev.elowar.com/):
+
+- “[Amiga® RKM Libraries: Exec Device I/O](http://amigadev.elowar.com/read/ADCD_2.1/Libraries_Manual_guide/node0299.html)”
+- “[Amiga® RKM Devices: Contents](http://amigadev.elowar.com/read/ADCD_2.1/Devices_Manual_guide/node0000.html)”
+
+## Compiling and testing
+The SD card support is in ROM.
+
+- With the [`master-new`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```
Comment (MBeijer) [inline docs/tasks/SD-card.md]:
```suggestion
- With the [`master`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```

Comment (MBeijer) [reply docs/tasks/SD-card.md]: perhaps remove this line entirely? both scripts are available though

File: docs/tasks/SD-card.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/v4-alynna/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it. There are some improvements pending merge at: [Some mkapollo.sh enhancements #27](https://github.com/ApolloTeam-dev/AROS/pull/27)
```
Comment (MBeijer) [inline docs/tasks/SD-card.md]:
```suggestion
- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/master/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it.
```

File: docs/tasks/Workbook.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+- “[Amiga® RKM Libraries: Workbench and Icon Library](http://amigadev.elowar.com/read/ADCD_2.1/Libraries_Manual_guide/node0237.html)”
+
+General documentation from AROS:
+
+- http://www.aros.org/documentation/developers/
+
+## Compiling and testing
+Workbook / `workbench.library` is in ROM.
+
+- With the [`master-new`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```
Comment (MBeijer) [inline docs/tasks/Workbook.md]:
```suggestion
- With the [`master`](https://github.com/ApolloTeam-dev/AROS) branch, which is the default, use `build_all.sh`.
```

File: docs/tasks/Workbook.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/v4-alynna/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it. There are some improvements pending merge at: [Some mkapollo.sh enhancements #27](https://github.com/ApolloTeam-dev/AROS/pull/27)
```
Comment (MBeijer) [inline docs/tasks/Workbook.md]:
```suggestion
- If you are using the [`mkapollo.sh`](https://github.com/ApolloTeam-dev/AROS/blob/master/mkapollo.sh) script, `./mkapollo.sh kernel` suffices to build it.
```

File: docs/tasks/Workbook.md [hunk 16]
```diff
@@ -0,0 +1,21 @@
+# Workbook
+
+Source code:
+
+- [`workbench/libs/workbench/`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/workbench/libs/workbench)
```
Comment (MBeijer) [inline docs/tasks/Workbook.md]:
```suggestion
- [`workbench/libs/workbench/`](https://github.com/ApolloTeam-dev/AROS/tree/master/workbench/libs/workbench)
```

File: docs/tasks/SD-card.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
+# SD card
+
+Source code:
+[`rom/devs/sdcard/`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/rom/devs/sdcard)
```
Comment (MBeijer) [inline docs/tasks/SD-card.md]:
```suggestion
[`rom/devs/sdcard/`](https://github.com/ApolloTeam-dev/AROS/tree/master/rom/devs/sdcard)
```

File: docs/tasks/SD-card.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+
+The SD card can only be used for reading files, not writing.
+
+The empty functions where the write operations should go are in
+[`rom/devs/sdcard/sdcard_ioops.c`](https://github.com/ApolloTeam-dev/AROS/tree/master-new/rom/devs/sdcard/sdcard_ioops.c)
```
Comment (MBeijer) [inline docs/tasks/SD-card.md]:
```suggestion
[`rom/devs/sdcard/sdcard_ioops.c`](https://github.com/ApolloTeam-dev/AROS/tree/master/rom/devs/sdcard/sdcard_ioops.c)
```

**Review by MBeijer — CHANGES_REQUESTED** [review]
Some things need to change due to `master-new` branch being renamed to `master` and `mkapollo.sh`-script being merged to master. :)

**Review by MBeijer — APPROVED** [review]

**Comment by MBeijer** [comment]
Awaiting your approval @ronybeck

**Comment by AlbertoGP** [comment]
There was today a post in the discord channel showing a roadmap for 2021:
https://discord.com/channels/726870908568076380/726871100243312650/822872181486649374

If someone else is tracking these things somewhere else, it might make more sense to put this information there, where it will be seen.

**Pull request closed without merging**